src/main/java/com/google/swarm/tokenization/common/Util.java

```diff
@@ -133,14 +115,22 @@ public static BufferedReader getReader(boolean customerSuppliedKey, String objec
 
 	}
 
+	public static String checkHeaderName(String name) {
```

Comment:
Yes, this was what I referring to in our previous talks. Bigquery does not allow spaces and other special characters. Should we expand the checker to remove all special characters?

Reply:
See below for a suggestion

Reply:
@theodoresiu  yes!  Can you create a PR after i merge?